web/assets/styles/themes/orange-tree.css

```diff
@@ -0,0 +1,70 @@
+/*
+ * Theme by Kuba Wolanin / github:kubawolanin
+ *
+ * Based on CometVISU/Knx template suggested by @Fejitatete
+ * on this thread:
+ * https://community.openhab.org/t/new-design-talent-wanted/29718
+ */
+:root {
+  --body-bg: #09120f;
+  --body-color: #a4a4a4;
+  --primary-color: hsl(29, 100%, 50%);
+  --box-bg: rgba(255, 255, 255, .01);
+  --header-bg: var(--body-bg);
+  --widget-design-bg: #191716;
+  --widget-text-color: #a4a4a4;
+  --widget-border-radius: 3px;
+  --widget-box-shadow: 0 1px 8px 0 rgba(0, 0, 0, .8), 0 1px 1px 0 rgba(0, 0, 0, .14), 0 2px 1px -1px rgba(0, 0, 0, .12);
+  --icon-color-filter: invert(263%) sepia(100%) hue-rotate(-42deg) saturate(100000%);
```

Comment:
I think the saturate is a little heavy :)
Looks like this:
![image](https://user-images.githubusercontent.com/2004147/27194559-4ca9e5ae-5203-11e7-9b08-0846838a3355.png)

`invert(1003%) sepia(100%) hue-rotate(-67deg) saturate(10000%)` looks like this:
![image](https://user-images.githubusercontent.com/2004147/27194541-33cdc49c-5203-11e7-907f-37350088fc73.png)